### DISCUSSION TITLE
[UI idea] Add some notification bar to notify users on new content or features

### DISCUSSION THREAD

**Discussion by TiBiBa** [hedyorg/hedy#5076] [Large Projects & Plans]
**Description**
It would be nice if we create some pop-up or notification that notifies users when we have deployed new content or new features. The admin should be able to "push" a message that users will see the first time they start a session (or either login). We do have to find a nice way to keep track of if logged in users have already seen the message.

This way we can notify them on the existence of the public profile, highscores and/or new adventures being added. In combination with this it might (not sure if this is within scope) be nice to enable teacher to sent notifications to their students.

**Comment by TiBiBa** [comment]
Maybe we can add this to the learn-more page as this is a bit static at the moment. I can understand returning users not visiting this page. We might be able to make it a bit more useful by adding some sort of "updates list" on which we post new updates and features. 

What do you think? @Felienne

**Comment by Felienne** [comment]
> **Description** It would be nice if we create some pop-up or notification that notifies users when we have deployed new content or new features. The admin should be able to "push" a message that users will see the first time they start a session (or either login). We do have to find a nice way to keep track of if logged in users have already seen the message.
> 
> This way we can notify them on the existence of the public profile, highscores and/or new adventures being added. In combination with this it might (not sure if this is within scope) be nice to enable teacher to sent notifications to their students.

Yes I think this is a lovely idea! Better than the newsletter that not a lot of people read (and that we can't easily localize). We already have the "inbox" for class invites, could we put it there? I think that is better than the learn more page. And if connected to an account it is not that hard, I guess to detect who has seen it.

**Comment by TiBiBa** [comment]
> > **Description** It would be nice if we create some pop-up or notification that notifies users when we have deployed new content or new features. The admin should be able to "push" a message that users will see the first time they start a session (or either login). We do have to find a nice way to keep track of if logged in users have already seen the message.
> > This way we can notify them on the existence of the public profile, highscores and/or new adventures being added. In combination with this it might (not sure if this is within scope) be nice to enable teacher to sent notifications to their students.
> 
> Yes I think this is a lovely idea! Better than the newsletter that not a lot of people read (and that we can't easily localize). We already have the "inbox" for class invites, could we put it there? I think that is better than the learn more page. And if connected to an account it is not that hard, I guess to detect who has seen it.

Sounds good! We currently only get these messages from the "invites" table, but I'm sure we can add another table called something like "updates" which we use to store updates. Or we manually add a message to our template and just check if the user has already seen it. Not sure what would be the best approach here: I like the idea of storing messages in a database so we can keep track, but this limits our options to style them.
